xml/System.Net/IPAddress.xml

```diff
@@ -2044,10 +2074,10 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">When this method returns, the IP address as a span of bytes.</param>
```

Comment:
```suggestion
        <param name="destination">When this method returns <see langword="true" />, receives the byte representation of the IP address, in network byte order.</param>
```

Reply:
I agree but shouldn't it be "contains" instead of "receives"? I also agree on the "When this method returns <see langword="true" />" part but that is not the pattern that [other](https://docs.microsoft.com/en-us/dotnet/api/system.int32.tryparse?view=netframework-4.8#System_Int32_TryParse_System_String_System_Int32__) `TryParse` APIs follow.

Reply:
"contains" to me is a little confusing because it sounds like a precondition to calling this method. "receives" tells me it will have the IP address written to it.

Reply:
@mairaw thoughts on true vs langword true?

Reply:
I don't think we need to say contains. So what happens when the method returns false?

Reply:
@mairaw if `TryWriteBytes` return false, it means that it didn't write into `destination`.

Reply:
> thoughts on true vs langword true?

@scalablecory Your suggestion is correct: we should use `<see langword="true" />` instead of just true.

+1 What @mairaw said: We should mention what the contents of `destination` will be if the method returns `false`.

Reply:
I'd be okay leaving without true or false as well. I just wouldn't say 
"When this method returns true, ..."

I'd simply say "When this method returns, "

People should be checking the result of the method anyway before trying to use these, I'd say